charts/puris/charts/backend/templates/deployment.yaml

```diff
@@ -48,6 +48,8 @@ spec:
               value: "{{ .Values.puris.datasource.password }}"
             - name: EDC_CONTROLPLANE_HOST
               value: "{{ .Values.puris.edc.controlplane.host }}"
+            - name: EDC_CONTROLPLANE_DATA_PATH
```

Comment:
Currently there is a medium finding regarding the container running with low uuid. This [tutorial ](https://docs.bridgecrew.io/docs/bc_k8s_37) explains how to solve it. It has already been present but is a low hanging fruit :)

You can find your user id in the respective Dockerfile (8877) :)

Reply:
Same applies to frontend BUT NGINX ships with a 101 user. For this case I should discuss it in the office hour.